ballerina/kafka_records.bal

```diff
@@ -116,6 +116,7 @@ public type AuthenticationConfiguration record {|
 # + excludeInternalTopics - Whether records from internal topics should be exposed to the consumer
 # + decoupleProcessing - Decouples processing
 # + validation - Configuration related to constraint validation check
+# + autoSeekOnValidationFailure - Enables seeking past any errornous records found when data-binding and validating constraints
```

Comment:
Enables seeking past any errornous records when data-binding and validating constraints

Reply:
Automatically seeks past the  errornous records in the event of an data-binding or validating constraints failure

Reply:
Updated